std/datetime/timezone.d

```diff
@@ -2007,6 +2008,14 @@ public:
         // Android concatenates all time zone data into a single file and stores it here.
         enum defaultTZDatabaseDir = "/system/usr/share/zoneinfo/";
     }
+    else version(Solaris)
+    {
+        /++
+            The default directory where the TZ Database files are. It's empty
+            for Windows, since Windows doesn't have them.
+          +/
```

Comment:
Why does this comment need to be copy-pasted for every platform?

Reply:
AFAICT just because the `version(TZDatabaseDir)` branch (which seems obsolete) doesn't start with the enum, but needs a prior import; otherwise, the comment could be placed once before the `version` branches. [Outside the scope of this PR though.]

Reply:
It was added a week ago: https://github.com/dlang/phobos/pull/5966

Reply:
FWIW we only support building the docs on `Posix` since 1.5 years, so adding comments in other branches isn't needed, but doesn't hurt either.
Anyhow @kinke said, it doesn't block this PR.

Reply:
The ddoc comment really should be put in a StdDdoc version block and removed from the other versions.

Reply:
I see no reason to put this off for later, I'd rather it were at the top with a StdDdoc version or just as a comment.  Just change it to say that Windows doesn't have one and TZDatabaseDir lets you set one at compile-time in an external file.

Reply:
Who can take care of this in a follow up?